test/controllers/journeys/adjustments/profitOrLoss/CheckNetProfitLossControllerSpec.scala

```diff
@@ -40,24 +40,27 @@ class CheckNetProfitLossControllerSpec extends ControllerSpec {
 
   def onPageLoadRequest = FakeRequest(GET, onPageLoad)
 
-  val profitOrLossCases = List(ProfitOrLoss.Profit) // TODO SASS-9032 replace with 'ProfitOrLoss.values' to test 'Loss' scenario
-  val defaultNetAmount  = BigDecimal(5000)          // TODO SASS-8626 remove and add tests for different API values
+  // TODO SASS-8626 replace with 'ProfitOrLoss.values' to test profit and loss scenarios when controller is dynamic
+  val profitOrLossCases = List(ProfitOrLoss.Loss)
+  val defaultNetAmount  = BigDecimal(-200) // TODO SASS-8626 remove and add tests for different API values
 
   "onPageLoad" - {
-    profitOrLossCases.foreach { profitOrLoss =>
-      s"when net $profitOrLoss" - {
+    "should return Ok and render correct view" - {
+      profitOrLossCases.foreach { profitOrLoss =>
         userTypeCases.foreach { userType =>
-          s"when user is an $userType, should return Ok and render correct view" in {
-            val application  = buildAppFromUserType(userType, Some(userAnswers))
-            implicit val msg = SpecBase.messages(application)
-            val result       = route(application, onPageLoadRequest).value
-            val netAmount    = formatMoney(defaultNetAmount, addDecimalForWholeNumbers = false)
+          s"when net $profitOrLoss and user is an $userType" in {
+            val application          = buildAppFromUserType(userType, Some(userAnswers))
+            implicit val msg         = SpecBase.messages(application)
+            val result               = route(application, onPageLoadRequest).value
+            val netAmount            = formatSumMoneyNoNegative(List(defaultNetAmount))
+            val netProfitOrLossTable = buildNetProfitOrLossTable(profitOrLoss, 3000, 0.05, -3100)
+            val additionsTable       = buildAdditionsTable(profitOrLoss, 0, -0.05, 100.20)
+            val deductionsTable      = buildDeductionsTable(profitOrLoss, 200, -200.1)
```

Comment:
These are the hardcoded values in the controller. When controller is dynamic in SASS-8626, the format of this test will change